forge-gui/res/cardsfolder/upcoming/flowstone_infusion.txt

```diff
@@ -0,0 +1,6 @@
+Name:Flowstone Infusion
+ManaCost:R
+Types:Instant
+A:SP$ Pump | ValidTgts$ Creature | NumAtt$ +2 | NumDef$ -2 | SpellDescription$ Target creature gets +2/-2 until end of turn.
+AI:RemoveDeck:Random
```

Comment:
Not sure this is needed here – similar things like Bedeck and Flowstone Armor don't have this AI indication